e2e/keywords/common.resource

```diff
@@ -13,7 +13,7 @@ Library             ../libs/keywords/workload_keywords.py
 *** Keywords ***
 Set test environment
```

Comment:
Probably you can make a more gradual setup or teardown for testsuite or test, so some resources are not required to clean up from test teardown but suite teardown. 

This can be figured out over time when having more tests.

Reply:
Tackle this later to see if there's any resources can be reused.